src/common/options/global.yaml.in

```diff
@@ -2135,6 +2135,16 @@ options:
   services:
   - mon
   with_legacy: true
+# actively release memory
+- name: mon_tcmalloc_aggressive_mem_decommit
```

Comment:
Why make this a `mon_` config? Why not for any daemon?

Reply:
Good point - I can see value for any daemon that actually uses tcmalloc, which includes OSDs.  Might want a note that RGW may not be in affected (unless it does use tcmalloc too)